api/src/org/apache/cloudstack/api/command/admin/vpc/VPCQuaggaConfigUpdateCmd.java

```diff
@@ -0,0 +1,204 @@
+    private String quaggaPassword;
+
+    @Parameter(name = QuaggaZoneConfig.s_ospfSuperCIDR, type = CommandType.STRING, required = false, description = "The super zone level CIDR for ospf enabled VPCs")
+    private String ospfSuperCIDR;
+
+    @Parameter(name = QuaggaZoneConfig.s_quaggaEnabled, type = CommandType.BOOLEAN, required = false, description = "The flag to enable or disable quagga for this zone")
+    private Boolean quaggaEnabled;
+
+    @Inject
+    public VpcProvisioningService _vpcProvSvc;
+
+    public Long getZoneid() {
+        return zoneid;
+    }
+
+    public void setZoneid(Long zoneid) {
+        this.zoneid = zoneid;
+    }
+
+    public Long getId() {
+        return zoneid;
+    }
+
+    public String getQuaggaProtocol() {
+        return quaggaProtocol;
+    }
+
+    public void setQuaggaProtocol(String quaggaProtocol) {
+        this.quaggaProtocol = quaggaProtocol;
+    }
+
+    public String getOspfArea() {
+        return ospfArea;
+    }
+
+    public void setOspfArea(String ospfArea) {
+        this.ospfArea = ospfArea;
+    }
+
+    public Short getQuaggaHelloInterval() {
+        return quaggaHelloInterval;
+    }
+
+    public void setQuaggaHelloInterval(Short quaggaHelloInterval) {
+        this.quaggaHelloInterval = quaggaHelloInterval;
+    }
+
+    public Short getQuaggaDeadInterval() {
+        return quaggaDeadInterval;
+    }
+
+    public void setQuaggaDeadInterval(Short quaggaDeadInterval) {
+        this.quaggaDeadInterval = quaggaDeadInterval;
+    }
+
+    public Short getQuaggaRetransmitInterval() {
+        return quaggaRetransmitInterval;
+    }
+
+    public void setQuaggaRetransmitInterval(Short quaggaRetransmitInterval) {
+        this.quaggaRetransmitInterval = quaggaRetransmitInterval;
+    }
+
+    public Short getQuaggaTransitDelay() {
+        return quaggaTransitDelay;
+    }
+
+    public void setQuaggaTransitDelay(Short quaggaTransitDelay) {
+        this.quaggaTransitDelay = quaggaTransitDelay;
+    }
+
+    public String getQuaggaAuthentication() {
+        return quaggaAuthentication;
+    }
+
+    public void setQuaggaAuthentication(String quaggaAuthentication) {
+        this.quaggaAuthentication = quaggaAuthentication;
+    }
+
+    public String getQuaggaPassword() {
+        return quaggaPassword;
+    }
+
+    public void setQuaggaPassword(String quaggaPassword) {
+        this.quaggaPassword = quaggaPassword;
+    }
+
+    public String getOspfSuperCIDR() {
+        return ospfSuperCIDR;
+    }
+
+    public void setOspfSuperCIDR(String ospfSuperCIDR) {
+        this.ospfSuperCIDR = ospfSuperCIDR;
+    }
+
+    public Boolean getQuaggaEnabled() {
+        return quaggaEnabled;
+    }
+
+    public void setQuaggaEnabled(Boolean quaggaEnabled) {
+        this.quaggaEnabled = quaggaEnabled;
+    }
+
+    @Override
+    public void execute() {
+        s_logger.info(this.toString());
```

Comment:
Why are logging the content of this request to ``INFO``?  It seems like it should be ``DEBUG``.